lib/readthis/cache.rb

```diff
@@ -214,6 +214,7 @@ def decrement(key, amount = 1, options = {})
     #
     def read_multi(*keys)
       options = merged_options(extract_options!(keys))
+      return {} if keys.empty?
```

Comment:
Unfortunately not because prior to that, the options hash is still part of `keys`.  The `keys.empty?` check needs to be after `extract_options!`